docs/docs/getting-started.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started
+
+## Prerequisites
+
+1. [Install Docker](https://docs.docker.com/engine/install/).
+
+## Initial Setup
+
+1. Clone this repository and `cd` into it.
+1. Set up your packages for the frontend: `cd frontend` and then `./run-after-changing-packages.sh`. This generates the `node_modules` folder and `package-lock.json` file in a Docker container and copies them locally. This is needed to run the site on your local machine.
+
+## Running a Local Instance of the Dashboard
+
+1. From the root of the repository, `cd frontend`. Then run `docker compose up` to start a local instance of the site.
+1. Navigate to `http://localhost:8080` in your web browser to see the site.
+
+    !!! tip
```

Comment:
Are you looking at it in the local doc server instance or just the GitHub rendering? On the local server it looks like this:

![image](https://github.com/Spelman-College/spelman-dashboard/assets/55670443/1931498e-6c96-46b7-87b3-bfe3a4c932fb)

(We should also refrain from using internal screenshot links for posterity since future contributors will not be able to view those. You can still use the screenshot tool, but you just need to copy/paste the image into here.)

Reply:
By the way, the `!!! tip` syntax is specific to MkDocs' Material theme: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
Got it, thanks! LGTM.